samples/basic/fade_led/README.rst

```diff
@@ -55,3 +25,20 @@ for the nrf52840dk_nrf52840 board:
    :board: nrf52840dk_nrf52840
    :goals: build flash
    :compact:
+
+Alternative configuration for board without on-board LED connected to PWM
+*************************************************************************
+
+If there is no on-board LED connected to a PWM configured PIN on your board,
+you can wire one manually and use a .overlay file to configure LED-PWM driver.
+An example of .overlay file is available in current folder and works with
+nucleo_f401re.
+
+You should build the sample with following command:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/basic/blink_led
+   :board: nucleo_f401re
+   :gen-args: -DDTC_OVERLAY_FILE="nucleo_f401re.overlay"
```

Comment:
This is not necessary. The build system picks up boards/BOARD.overlay automagically.